components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```
@@ -4139,4 +4194,27 @@ public static NativeObject jsFunction_getDocument(Context cx, Scriptable thisObj
 
         return data;
     }
+
+    /**
+     * This method gets the group Id of the current logged in user.
+     * @param cx Rhino Context
+     * @param thisObj Scriptable object
+     * @param args Passing arguments
+     * @param funObj Function object
+     * @return String group id. 
+     * 
+     */
+    public static String jsFunction_getGroupIds(Context cx, Scriptable thisObj, Object[] args, Function funObj) {
+        String response = (String) args[0];
+        APIConsumer consumer = getAPIConsumer(thisObj);
+        String groupId = null;
+        try {
+            groupId = consumer.getGroupIds(response);
+        } catch (APIManagementException e) {
+            log.error("Error occurred while getting group id");
```

put a comment,mentioning the reason,why swallowing the exception..